### ISSUE TITLE
OTP

### ISSUE THREAD

**Issue by anonymous1184** [mssun/passforios#439]
Hi!

Any plan to send the OTP as notification and/or copying it to clipboard? It's a super-useful feature!

Thanks for your time and efforts with the app.

**Comment by mssun** [comment]
Hi, what do you mean by sending the OTP as a notification?

Right now, you can manually copy the OTP by tapping the generated password.

**Comment by anonymous1184** [comment]
Yes, I use `pass` because is wonderful how much I can integrate it with so many workflows across Windows/macOS and a myriad of Linux VMs, however iOS is a whole different story. I'd love pass for iOS had this feature (like MYKI), because it removes the need of going to the app (or use another). The most easy approach will be to send to clipboard the code and a wonderful addition will be to also a notification sent (it wouldn't hurt a space in between for the brain to group the numbers).

![IMG_4129](https://user-images.githubusercontent.com/53758552/103393031-4ece2b80-4ae6-11eb-9fa3-c31bdbe025e0.jpeg)

Again, thanks!

**Comment by blockloop** [comment]
@mssun copying by navigating to the app is a slow process. I think a setting to automatically copy OTP to the clipboard after autofill would be ideal. This is what [browserpass has implemented](https://github.com/Preston12321/browserpass-extension/commit/52c9987ef8c8ac8b9a89565bcf5907e57a3e93a7) and it's also what other mobile apps like 1Password use. I would love to see this feature in your app.
